testdata/baselines/reference/submodule/compiler/commentsemitComments.js.diff

```diff
@@ -20,13 +20,12 @@
          constructor(x) {
              this.x = x;
          }
-@@= skipped -56, +57 lines =@@
-         x: number;
-         constructor(x: number);
-     }
-+    /// module m2
-     namespace m2 {
-     }
- }
-+/// this is x
- declare var x: any;
\ No newline at end of file
+@@= skipped -43, +44 lines =@@
+     /**indexer property*/
+     [a: number]: string;
+     /** function property;*/
+-    myFoo(/*param prop*/ a: number): string;
++    myFoo(a: number): string;
```

Comment:
Interestingly, this actually appears to be an improvement. Not sure why this non-JSDoc comment was preserved here.